tests/integration/__init__.py

```diff
@@ -195,7 +195,7 @@ def get_supported_protocol_versions():
 `   """
     version = Version(CASSANDRA_VERSION)
```

Comment:
This looks good to me, could we take the chance to update this and add:
```python
if version >= Version('4.0'):
    return (3, 4, 5)
if version >= Version('3.11'):
    return (3, 4)
...

Reply:
can do :+1: